src/board.cpp

```diff
@@ -0,0 +1,166 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include "board.h"
+#include "gui.h"
+
+Board::Board() : Board(9, false) {}
+
+Board::Board(size_t size, bool cpu) {
+  isCpu = cpu;
+  p1Score = 0;
+  p2Score = 0;
+  boardSize = size;
+  for (int i = 1; i <= size; i++) {
```

Comment:
1 indexed?

Reply:
My reasoning is that it makes it easier to store the moves and handle user input (which is 1-indexed). Plus it's not a vector so index doesn't matter too much.